src/firmware.cpp

```diff
@@ -2,51 +2,33 @@
 
 #include <Adafruit_PWMServoDriver.h>
 
-// Multiplying by this converts round-trip duration in microseconds to distance to object in millimetres.
-static const float ULTRASOUND_COEFFICIENT = 1e-6 * 343.0 * 0.5 * 1e3;
-
-static const String FIRMWARE_VERSION = "SourceBots PWM/GPIO v0.0.1";
-
-typedef String CommandError;
+static Adafruit_PWMServoDriver SERVOS = Adafruit_PWMServoDriver();
 
-static const CommandError OK = "";
+typedef String CommandResponse;
+static const CommandResponse OK = "";
 
 #define COMMAND_ERROR(x) ((x))
```

Comment:
This #define just seems to obfiscate the output as it is just a string, might be worth it if the return is a struct or similar and error code is also reported.